music.js

```diff
@@ -1,17 +1,40 @@
 // Define the Artist constructor function here, with a 'name' property that can be set at instantiation,
 // and a 'songs' property that is an empty array to start
-function Artist() {
-
+function Artist(name) {
+  this.name = name;
+  this.songs = [];
 }
 
+
 // Define the Song constructor function here, with 'title' and 'artist' properties that can be set at instantiation,
 // and a 'playCount' property that is an empty array to start
-function Song() {
+function Song(title, artist) {
+  this.title = title;
+  this.artist = artist;
```

Comment:
If you had wanted to safeguard your code to make *sure* that the artist variable coming in was the right type of Object before using it, what could you do? (hint: remember the `typeof` operator).